api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -63,7 +63,7 @@ private ClusterRequest createFirecloudClusterRequest(String userEmail) {
   public Cluster createCluster(String googleProject, String clusterName, String userEmail) {
     ClusterApi clusterApi = clusterApiProvider.get();
     return retryHandler.run((context) ->
-        clusterApi.createCluster(googleProject, clusterName, createFirecloudClusterRequest(userEmail)));
+        clusterApi.createClusterV2(googleProject, clusterName, createFirecloudClusterRequest(userEmail)));
```

Comment:
We also want to pass our oauth client ID as the `defaultClientID` when creating the cluster, but that could be a follow-up.

Reply:
Looked into this, and saw it is a bit more complicated than just adding it to the `createFirecloudClusterRequest` function, so punting for now.